pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-inject-parameters.adoc

```diff
@@ -0,0 +1,305 @@
+= Injecting Parameters
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+PDK provides the following parameters that you can inject into the `#[entrypoint]` configuration function:
+
+* `Configuration`: Provides the policy's configuration parameters. For information about how to define configuration parameters, see xref:policies-pdk-create-schema-definition.adoc[].
+* `Metadata`: Provides metadata about the Policy, the Flex instance, the API instance, and the Anypoint Organization. For more information about the metadata provided, see <<metadata>>.
+* `HttpClient`: Enables the policy to make HTTP calls. For more information about how to make HTTP calls from the policy, see xref:policies-pdk-configure-features-http-request.adoc[].
```

Comment:
"Enables" implies that the value of `HttpClient` is a boolean. Is this the case? If so does this need to be specified?